### PR TITLE
[onert/train] Initialize class field for train

### PR DESCRIPTION
**Pull request by hseok-oh**
This commit initializes ElementwiseActivationLayer and TrainingInfo class fields.

ONE-DCO-1.0-Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Review by chunseoklee — APPROVED** [review]

File: runtime/onert/core/include/compiler/train/TrainingInfo.h [hunk 1]
```diff
@@ -56,9 +56,9 @@ class TrainingInfo
   }
 
 private:
-  LossInfo _loss_info;
-  ir::train::OptimizerInfo _optimizer_info;
-  uint32_t _batch_size;
+  LossInfo _loss_info{ir::operation::Loss::Type::MEAN_SQUARED_ERROR};
+  ir::train::OptimizerInfo _optimizer_info{ir::train::OptimizerCode::Invalid, 0};
+  uint32_t _batch_size = 0;
```
Comment (ragmani) [inline runtime/onert/core/include/compiler/train/TrainingInfo.h:61]: Just out of curiosity, what is the difference between using uniform initialization and assignment?

**Review by ragmani — APPROVED** [review]
LGTM

**Comment by glistening** [comment]
@hseok-oh Could you please fix a typo in commit title?

File: runtime/onert/core/include/compiler/train/TrainingInfo.h (hunk 1 above)
Comment (hseok-oh) [reply runtime/onert/core/include/compiler/train/TrainingInfo.h:61]: Maybe no difference.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**